packages/account/src/Components/api-token/api-token.tsx

```diff
@@ -1,36 +1,68 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 import classNames from 'classnames';
-import { Formik, Form, Field } from 'formik';
+import { Formik, Form, Field, FormikValues, FormikErrors, FieldProps } from 'formik';
 import { Timeline, Input, Button, ThemedScrollbars, Loading } from '@deriv/components';
 import InlineNoteWithIcon from '../inline-note-with-icon';
 import { isDesktop, isMobile, getPropertyValue, useIsMounted } from '@deriv/shared';
 import { localize } from '@deriv/translations';
 import LoadErrorMessage from 'Components/load-error-message';
-import ApiTokenArticle from './api-token-article.jsx';
-import ApiTokenCard from './api-token-card.jsx';
-import ApiTokenFooter from './api-token-footer.jsx';
-import ApiTokenOverlay from './api-token-overlay.jsx';
-import ApiTokenTable from './api-token-table.jsx';
+import ApiTokenArticle from './api-token-article';
+import ApiTokenCard from './api-token-card';
+import ApiTokenFooter from './api-token-footer';
+import ApiTokenOverlay from './api-token-overlay';
+import ApiTokenTable from './api-token-table';
 import ApiTokenContext from './api-token-context';
 
 const MIN_TOKEN = 2;
 const MAX_TOKEN = 32;
 
-const ApiToken = ({ footer_ref, is_app_settings, is_switching, overlay_ref, setIsOverlayShown, ws }) => {
+type AptTokenState = {
+    api_tokens: NonNullable<import('Types').TToken[]>;
+    is_loading: boolean;
+    is_success: boolean;
+    is_overlay_shown: boolean;
+    error_message: string;
+    show_delete: boolean;
+    dispose_token: string;
+    is_delete_loading: boolean;
+    is_delete_success: boolean;
+};
+
+export type TApiToken = {
+    footer_ref: Element | DocumentFragment | undefined;
+    is_app_settings: boolean;
+    is_switching: boolean;
+    overlay_ref:
+        | undefined
+        | ((...args: unknown[]) => unknown)
+        | import('prop-types').InferProps<{
+              current: import('prop-types').Requireable<unknown>;
+          }>;
+    setIsOverlayShown: (is_overlay_shown: boolean | undefined) => void;
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    ws: any;
```

Comment:
Is there any plan for defining types for **WS**? because I saw many of them in the project